lib/Serverless.js

```diff
@@ -15,6 +16,9 @@ const Service = require('./classes/Service');
 const Variables = require('./classes/Variables');
 const ServerlessError = require('./classes/Error').ServerlessError;
 const Version = require('./../package.json').version;
+const isStandaloneExecutable = require('./utils/isStandaloneExecutable');
+
+const frameworkUpdateCommands = new Set(['upgrade', 'uninstall']);
```

Comment:
Any reason to call this `upgrade` rather than `update`? I do see the inspiration from package managers, just asking about the terminology...

Just a nit regarding the variable name: Why is `uninstall` an `UpdateCommand`?

Reply:
> Any reason to call this `upgrade` rather than `update`

Yes, it's purely to be on pair with `choco` or `brew`.

_upgrade_ probably feels better for operation when binary is replaced.

Reply:
Makes sense :+1:

Reply:
> Just a nit regarding the variable name: Why is uninstall an UpdateCommand?

I've updated the name to `installationMaintananceCommands`